src/parser/tests/parser.rs

```diff
@@ -127,6 +127,67 @@ fn parse_variable_tag_lit_math_expression() {
     );
 }
 
+#[test]
+fn parse_variable_tag_concat_strings() {
+    let ast = parse("{{ \"a\" ~ \"b\" }}").unwrap();
+
+    assert_eq!(
+        ast[0],
+        Node::VariableBlock(Expr::new(ExprVal::Concat(Concat {
+            lhs: Box::new(Expr::new(ExprVal::String("a".to_string()))),
+            rhs: Box::new(Expr::new(ExprVal::String("b".to_string()))),
+        }))),
```

Comment:
in the case of 2 literal strings concat together, the parser should just create a String rather than a Concat since we can figure out the result at parsing time. The only time a Concat should be created is if one of the side is an ident since we can't resolve it at parsing time